src/blister.c

```diff
@@ -1443,28 +1436,28 @@ static Obj FuncUNITE_BLIST_LIST(Obj self, Obj list, Obj blist, Obj sub)
         k = 1;
         block = 0;
         bit   = 1;
+        Obj elm = 0, prev;
         for ( l = 1; l <= lenList; l++ ) {
-
+            prev = elm;
+            elm = ELM_LIST(list, l);
             /* test if <list>[<l>] is in <sub>                             */
-            if ( l == 1 || LT(CONST_ADDR_OBJ(list)[l-1],CONST_ADDR_OBJ(list)[l]) ){
-                while ( k <= lenSub
-                     && LT(CONST_ADDR_OBJ(sub)[k],CONST_ADDR_OBJ(list)[l]) )
+            if (l == 1 || LT(prev, elm)) {
+                while (k <= lenSub && LT(ELM_PLIST(sub, k), elm))
```

Comment:
Note that this code was broken if `list` contains holes, as then we were passing NULL to `LT`, which would result in nonsense or crashes. Now we get a proper error, due to the use of `ELM_LIST`. (I guess I should add a test case for that, too). 

The new code will still do weird things if `sub` contains holes, but that means it is not worse than it was before.

Reply:
Is there a good reason not to just remove all the `PLAIN_LIST`s and make everything ELM_LIST variants? (or are you working through in phases)?

Reply:
Good question, I forgot to remove the last one. But I corrected that, so all *direct* `PLAIN_LIST` calls in that function are gone. Mind you, it is still indirectly called via `IsSet`; changing that is